scripts/verify-test-resource.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -o nounset # don't allow use of undefined variables
+set -o pipefail # if any single command in a pipe chain exits non-zero, set the exit status to that
+
+# Set environment variable for GitHub action to recognise colours
+export TERM=xterm
+
+# Colours for stdout
+RED=$(tput setaf 1)
+GREEN=$(tput setaf 2)
+WHITE=$(tput setaf 7)
+NC=$(tput sgr0)
+RED_BG=$(tput setab 1)
+
+# Path to test resources
+TEST_RESOURCES_PATH=src/test/resources
+
+# Temporary directory for saving compiled classes
+TEMP_DIR_COMPILED_RESOURCES=$(mktemp -d)
+echo "Created $TEMP_DIR_COMPILED_RESOURCES for saving compiled files temporarily."
+echo -e "It will be removed after the script has executed with any exit code.\n"
```

Comment:
As you don't have `errexit` enabled, you must check if the temporary directory was actually created. If it wasn't, you might have just an empty string in `TEMP_DIR_COMPILE_RESOURCES`, and then doing `rm -rf "$TEMP_DIR_COMPILE_RESOURCES"` wouldn't be great...

Reply:
Okay, this makes sense. If the directory is not created, I will exit the script with status code `1`.